.evergreen/config.yml

```diff
@@ -776,8 +781,13 @@ functions:
       shell: bash
       script: |-
         set -o errexit
+        # Only run if a load balancer was started.
+        if [ -z "${SINGLE_MONGOS_LB_URI}" ]; then
+            echo "OK - no load balancer running"
+            exit 0
+        fi
         export DRIVERS_TOOLS=./drivers-evergreen-tools
-        export MONGODB_URI=${MONGODB_URI}
+        export MONGODB_URI="foo" # TODO: DRIVERS-1833 remove this.
```

Comment:
This can be removed once https://github.com/mongodb-labs/drivers-evergreen-tools/pull/159 is merged.